feat: adding recipes tracker workshop (#56456)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66ff02792b3042a2b811f475.md

---
id: 66ff02792b3042a2b811f475

> title: Step 3

Author: Can the step go in after all the objects and their properties have been filled in?

> title: Step 3

Reviewer: sure

> title: Step 3

Author: It's there now

challengeType: 1
dashedName: step-3
---

# --description--

Add the remaining properties from the following list to the `recipe1` object:

| Key | Value   |
| ----------- | ------- |
|`cookingTime`|`22`|
|`totalIngredients`|`null`|
|`difficultyLevel`|`""`|
|`averageRating`|`null`|

> The properties with `null` and empty array values will have items pushed into them later after you calculate them.

Reviewer 2: ```suggestion
The properties with `null` and empty string values will be updated later after you calculate them.
```
